matrix/main.py

```diff
@@ -44,7 +45,10 @@ def setup(matrix, args=None):
     parser.add_argument("-i", "--interval", default=5.0, type=float)
     parser.add_argument("-p", "--path", default=Path.cwd() / "tests",
                         type=Path)
-    parser.add_argument("config_file", type=open)
+    parser.add_argument("-c", "--config-file",
+                        default=resource_filename(__name__, 'matrix.yaml'))
```

Comment:
If it's not bundled as part of the Matrix, how does someone who wants to run the standard suite against their bundle under development run it?

I was also under the impression that we wanted to merge the in-bundle suite with the standard suite, rather than running it as a separate instance of Matrix nor in place of the standard suite.  So, I'm not sure what the benefit of the she-bang line for matrix tests would be.